_studio/mfx_lib/genx/CMakeLists.txt

```diff
@@ -77,6 +77,33 @@ if(MFX_ENABLE_KERNELS)
   set( sources "" )
   set( sources.plus "" )
 
+
+  # copy kernels
```

Comment:
why you moved this up? in that way it is more difficult to track file renames. Please, restore prev. order of ASC/copy/MCTF sections.

Reply:
ok